defaults/main.yml

```
@@ -61,6 +60,7 @@ hub_configuration:
   # hostname can contain multiple hostnames but we only need one
   # for most following options
   primary_hostname: "{{ hostname.split(',')[0] | trim }}"
+  secondary_hostnames: "{{ hostname.split(',') | difference(hostname.split(',')[0] | trim) }}"
```

don't know what `trim` does on an array but I suspect it not to work. Also could be simplified to:

```suggestion
  secondary_hostnames: "{{ hostname.split(',')[1:] | trim) }}"
```

basically says: take everything the array from index 1 until the end. Learn more about [slicing](https://stackoverflow.com/a/509295/1142028)

---

what happens if there is no index 1? My version dont throws an error but just gives an empty string

---

emptyyy array